src/netstandard/pkg/shims/netstandard/System.Buffers.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>System.Buffers</AssemblyName>
+    <AssemblyVersion>4.0.2.0</AssemblyVersion>
```

Comment:
Do all these match exactly what was in packages or are they higher?

Reply:
These match what was in the packages

Reply:
Here's what we did: https://github.com/dotnet/standard/commit/f9f0e4c6c9430d345d327a93d20fd89befcd49d1

We bumped the 3rd portion of the version so that it would be greater than any serviced version (where we increment the 4th part).

Reply:
Just pushed a commit to resolve this